app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+<script>
+    class MyAwesomeButton extends Polymer.Element {
+      static get is() {
+        return "my-awesome-button";
+      }
+    }
+    customElements.define(MyAwesomeButton.is, MyAwesomeButton);
+</script>
+
+## Third-party libraries that modify local DOM {#scope-subtree}
+
+If you are using a third-party library that adds local DOM nodes to your
+Polymer element, you may notice that styles on the element do not update
+properly.
+
+The correct way to add DOM nodes to a Polymer element's local DOM is via
+the Polymer [DOM API](local-dom#dom-api). This API lets you manipulate
+nodes in a way that respects the local DOM and ensures that styles are
+updated properly.
+
+When using third-party libraries that **do not use** the Polymer DOM
+API, use the [`scopeSubtree`](/2.0/docs/api/Polymer.Base#method-scopeSubtree)
+method to apply proper CSS scoping to a node and all of its descendants.
+
+1.  Create a container node inside your element's local DOM, and have your
+    third-party library create DOM under that container node.
+
+    ```html
+    <dom-module is="x-example">
+      <template>
+        <div id="container">
+        </div>
+      </template>
+    </dom-module>
+    ```
+	
+2.  Call `scopeSubtree` on the container node.
+
+    ```js
+    ready: function() {
+      this.scopeSubtree(this.$.container, true);
+    }
+    ```
+
+    `containerNode` is the root node of the tree you wish to scope. Setting
+    the second argument to `false` scopes the specified node and descendants
+    **once.** Setting it to `true` enables a mutation observer that applies CSS
+    scoping whenever `containerNode` or any of its descendants are modified.
+
+**Not for use on Polymer elements.** If the subtree that you scope
+contains any Polymer elements with local DOM, `scopeSubtree` will
+cause the descendants' local DOM to be styled incorrectly.
+{.alert .alert-error}
+
+## Styling at the document level
+	
+Browsers that implement the current Shadow DOM v1 specifications will automatically encapsulate styles, scoping them to the elements in which they were defined.
+
+Some browsers have not implemented the Shadow DOM v1 specifications. To make sure your apps and elements display correctly in these browsers, you'll need to use `custom-style` to ensure that styling information doesn't "leak" into the local DOM of your elements.
+
+The `custom-style` extension enables a set of polyfills that ensure that styles in your apps and elements behave as you would expect from the Shadow DOM v1 specifications, even in browsers that don't implement these specifications.
+
+`custom-style` is an extension of the native `<style>` element. To ensure that your styles behave according to the Shadow DOM v1 specifications in all browsers, use `custom-style` when you define *document-level* styles.
+
+*Note: You should only use `custom-style` to define styles for the main document. To define styles for an element's local DOM, just use a `<style>` block.*
```

Comment:
We need to add that this is optional and it needs to be imported separately (unless you're importing the full `polymer.html`). 

You could show an example of an `index.html` that imports the polyfills and then imports `polymer/lib/elements/custom-style.html` (since it's not _defining_ any Polymer elements, 
it doesn't need to import polymer.html or polymer-element.html). 

It would then import the custom element, and use a custom-style to apply styles... make sense?